pulumi/tests/test_network.py

```diff
@@ -0,0 +1,70 @@
+import os
+import pulumi
+import sys
+import unittest
+from unittest.mock import MagicMock, patch
+
+
+class MyMocks(pulumi.runtime.Mocks):
+    def new_resource(self, args: pulumi.runtime.MockResourceArgs):
+        outputs = args.inputs
+        return [args.name + '_id', outputs]
+
+    def call(self, args: pulumi.runtime.MockCallArgs):
+        if args.token == "aws:index/getAvailabilityZones:getAvailabilityZones":
+            return {
+                "names": ["a", "b", "c"],
+                "state": "available"
+            }
+        # This is necessary since infra.config calls this
+        if args.token == "aws:index/getCallerIdentity:getCallerIdentity":
+            return {
+                "account_id": "000000000000"
+            }
+        return {}
+
+pulumi.runtime.set_mocks(MyMocks())
+
+# Environment variable is required on import so we can't monkeypatch. Set it here for now
+os.environ["GRAPL_OPERATIONAL_ALARMS_EMAIL"] = "fake-ops-email@fake.domain"
+sys.modules['infra.network.infra.config'] = MagicMock()
```

Comment:
Log-term we'll need to either make infra.config side-effect free on import OR create a test runner that sets the environment variables properly.